src/current/v24.3/ui-logical-data-replication-dashboard.md

```diff
@@ -0,0 +1,107 @@
+
+The graph shows the difference in commit times between the source cluster and the destination cluster.
+
+Metric | CockroachDB Metric Name | Description
+-------|-------------------------|-------------
+p50,<br>p99 | `logical_replication.commit_latency-p50`,<br>`logical_replication.commit_latency-p99` | Event commit latency: a difference between event MVCC timestamp and the time it was flushed into disk. If we batch events, then the difference between the oldest event in the batch and flush is recorded.
+
+## Replication Lag
+
+The graph shows the age of the oldest row on the source cluster that has yet to replicate to the destination cluster.
+
+Metric | CockroachDB Metric Name | Description
+-------|-------------------------|-------------
+Replication Lag | `logical_replication.replicated_time_seconds` | The replicated time of the logical replication stream in seconds since the unix epoch.
+
+## Row Updates Applied
+
+The graph shows the rate at which row updates are applied by all logical replication jobs.
+
+Metric | CockroachDB Metric Name | Description
+-------|-------------------------|-------------
+Row Updates Applied | `logical_replication.events_ingested` | Events ingested by all replication jobs.
+Row Updates sent to DLQ | `logical_replication.events_dlqed` | Row update events sent to DLQ.
+
+## Logical Bytes Received
+
+- In the cluster view, the graph shows the rate at which the logical bytes (sum of keys + values) are received by all logical replication jobs across all nodes.
+- In the node view, the graph shows the rate at which the logical bytes (sum of keys + values) are received by all logical replication jobs on the node.
+
+Metric | CockroachDB Metric Name | Description
+-------|-------------------------|-------------
+logical bytes | `logical_replication.logical_bytes` | Logical bytes (sum of keys + values) received by all replication jobs.
+
+## Batch Application Processing Time: 50th percentile
+
+- In the cluster view, the graph shows the 50th percentile in the time it takes to write a batch of row updates across all nodes.
+- In the node view, the graph shows the 50th percentile in the time it takes to write a batch of row updates on the node.
+
+Metric | CockroachDB Metric Name | Description
+-------|-------------------------|-------------
+processing time | `logical_replication.batch_hist_nanos-p50` | Time spent flushing a batch.
+
+
+## Batch Application Processing Time: 99th percentile
+
+- In the cluster view, the graph shows the 99th percentile in the time it takes to write a batch of row updates across all nodes.
+- In the node view, the graph shows the 99th percentile in the time it takes to write a batch of row updates on the node.
+
+Metric | CockroachDB Metric Name | Description
+-------|-------------------------|-------------
+processing time | `logical_replication.batch_hist_nanos-p99` | Time spent flushing a batch.
+ 
+## DLQ Causes
+
+The graph shows the reasons why events were sent to the DLQ.
```

Comment:
I would maybe expand the DLQ acronym here again and maybe link to https://www.cockroachlabs.com/docs/v24.3/manage-logical-data-replication.html#dead-letter-queue-dlq